BraintreeDropIn/BTPaymentSelectionViewController.m

```diff
@@ -187,9 +187,16 @@ - (void)configurationLoaded:(__unused BTConfiguration *)configuration error:(NSE
                 }
             }
 
-            // Always add Cards
-            [activePaymentOptions addObject:@(BTUIKPaymentOptionTypeUnknown)];
-            
+            NSArray *supportedCardTypes = [self.configuration.json[@"creditCards"][@"supportedCardTypes"] asArray];
+            for (NSString *supportedCardType in supportedCardTypes) {
+                BTUIKPaymentOptionType paymentOptionType = [BTUIKViewUtil paymentOptionTypeForPaymentInfoType:supportedCardType];
+                if ([BTUIKViewUtil isPaymentOptionTypeACreditCard:paymentOptionType]) {
```

Comment:
Is there any other place we would use this new method? Or can we simplify this logic to:

```objc
if ([supportedCardTypes count] > 0) {
     [activePaymentOptions addObject:@(BTUIKPaymentOptionTypeUnknown)];
}
```